packages/next/src/build/webpack/loaders/next-metadata-route-loader.ts

```diff
@@ -79,6 +78,19 @@ export async function GET() {
 `
 }
 
+function getDynamicImageRouteCode(resourcePath: string) {
+  return `\
+import { NextResponse } from 'next/server'
+import handler from ${JSON.stringify(resourcePath)}
+
+export async function GET(req, ctx) {
+  const res = await handler({ params: ctx.params })
+  res.headers.set('Cache-Control', 'public, max-age=0, must-revalidate')
```

Comment:
Why does this override cache-control?

Reply:
The url pathname is fixed, but og package so far gives a default long cache header for production. To avoid the changes being long cached i changed it to must-revalidate, like other dynamic routes